### ISSUE TITLE
can I convert rtf containing inline images to html using this library?

### ISSUE THREAD

**Issue by milankamboya** [sbraconnier/jodconverter#155]
I am using com.artofsolving > jodconverter to convert rtf text to html in our use case.
But it is not converting rtf text with inline images to html.
So I want to use this but will it convert rtf text with inline images to html?

**Comment by sbraconnier** [comment]
If you can do it with Libre Office without JODConverter, then it is possible to do it with JODConverter. Give it a try and let me know if it worked. If not, I may be able to help you with a little more information and I will need an input file to work with.

**Comment by milankamboya** [comment]
@sbraconnier 
Sorry for late thing.
I am checking this issue after long time but still I have issue.

**I want to ask that:**
**Question** : can we convert RTF text which contains inline images to HTML ?
Here RTF text means, we have used inline rtf editor through which user writes text and it stores into database in text/string as hexadecimal values.

Example as simple RTF text:
Actual text: 
"text object"

Stored value in DB :
`7B5C727466315C616E73695C64656666305C7563315C616E7369637067313235325C6465667461623732307B5C666F6E7474626C7B5C66305C666E696C5C66636861727365743120536F6C756D696E6120416E736920593B7D7B5C66315C666E696C5C666368617273657431205461686F6D613B7D7B5C66325C666E696C5C6663686172736574322057696E6744696E67733B7D7B5C66335C666E696C5C6663686172736574322053796D626F6C3B7D7D7B5C636F6C6F7274626C5C726564305C677265656E305C626C7565303B5C7265643235355C677265656E305C626C7565303B5C726564305C677265656E3132385C626C7565303B5C726564305C677265656E305C626C75653235353B5C7265643235355C677265656E3235355C626C7565303B5C7265643235355C677265656E305C626C75653235353B5C7265643132385C677265656E305C626C75653132383B5C7265643132385C677265656E305C626C7565303B5C726564305C677265656E3235355C626C7565303B5C726564305C677265656E3235355C626C75653235353B5C726564305C677265656E3132385C626C75653132383B5C726564305C677265656E305C626C75653132383B5C7265643235355C677265656E3235355C626C75653235353B5C7265643139325C677265656E3139325C626C75653139323B5C7265643132385C677265656E3132385C626C75653132383B5C726564305C677265656E305C626C7565303B5C726564305C677265656E305C626C7565303B7D5C7770707268656164666F6F74315C70617065727731323234305C70617065726831353834305C6D6172676C313838305C6D61726772313838305C6D61726774313434305C6D61726762313434305C686561646572793732305C666F6F746572793732305C656E646E686572655C7365637464656661756C74636C7B5C2A5C67656E657261746F72205750546F6F6C735F372E3030303B7D7B5C706C61696E5C66315C667331385C636631362074657874206F626A6563745C7061720D0A5C706172645C706C61696E5C706C61696E5C66315C667331385C636631365C7061720D0A5C706172645C706C61696E5C706C61696E5C667332385C7061720D0A7D7D`

Is it possible to do with this repository?

**What I tried:**
I have created one rtf file using open-office which contains some text and images.
Then I have used this repo's sample example to pas it as multipart file and it converts to HTML properly.
But with same internal logic (which has been used in sample example to convert), I have tried to convert RTF hexadecimal string to HTML. But it is not working.

Any idea? what can be done?
